applications/HDF5Application/tests/test_utils.cpp

```diff
@@ -1,20 +1,222 @@
-#include "tests/test_utils.h"
-
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Michael Andre, https://github.com/msandre
+//                   Suneth Warnakulasruiya
+//
+
+// System includes
 #include <algorithm>
+#include <type_traits>
 
+// Project includes
 #include "containers/array_1d.h"
+#include "includes/kratos_components.h"
 #include "includes/kratos_parameters.h"
 #include "includes/model_part.h"
-#include "includes/kratos_components.h"
+#include "includes/parallel_environment.h"
 #include "testing/testing.h"
+
+// Application includes
 #include "custom_io/hdf5_file.h"
-#include "custom_utilities/registered_component_lookup.h"
+
+// Include base h
+#include "tests/test_utils.h"
 
 namespace Kratos
 {
 namespace Testing
 {
 
+
+namespace TestModelPartFactoryHelperUtilities
```

Comment:
Test

Factory

Helper

Utility

please find a better name for this namespace. How about `HDF5TestUtilities`?


```suggestion
namespace TestModelPartFactoryHelperUtilities
```

Reply:
done :)